frontend/src/services/fileService.ts

```diff
@@ -64,11 +67,28 @@ export async function uploadFiles(files: FileList): Promise<UploadResult> {
 }
 
 export async function listFiles(path: string = "/"): Promise<string[]> {
-  try {
-    const encodedPath = encodeURIComponent(path);
-    const data = await request(`/api/list-files?path=${encodedPath}`);
-    return data as string[];
-  } catch (error) {
-    return [];
+  const data = await request(
```

Comment:
why don't we use the `try...catch` pattern?

Reply:
That's because @amanape had recommended to let the exceptions be caught at a higher level, not here.
But I see that I missed to re-add the encodedURIComponent usage. 😢 
Will do that in a minute.